test/query_component_test.dart

```diff
@@ -277,17 +264,17 @@ void main() {
         ];
       });
 
-      tearDown(() => clearInteractions(mockQueryService));
+      tearDown(() => clearInteractions(mockGoogleApiDart));
 
       test('no advertiser id disables the populate button', () async {
         await queryComponentPO.clickPopulate();
-        verifyNever(mockQueryService.execDV3Query(any, any, any, any, any));
+        verifyNever(mockGoogleApiDart.request(any));
       });
 
       test('non-integer advertiser id disables the populate button', () async {
         await queryComponentAccordionPO.typeAdvertiserId('wrong-format-id');
         await queryComponentPO.clickPopulate();
-        verifyNever(mockQueryService.execDV3Query(any, any, any, any, any));
+        verifyNever(mockGoogleApiDart.request(any));
       });
 
       test('highlighting unchecked invokes populate(multiple-IO, false)',
```

Comment:
are you testing the checkbox or all of the queries? I suggest you confirm the queries once in one test, and in 2 other test confirm the checkbox behavior. this way there are fewer duplicated asserts and it is more clear what the test is testing.